spec/factories/booth_request.rb

```diff
@@ -2,7 +2,7 @@
   factory :booth_request do
     booth
     user
-    role 'responsible'
+    role %w[submitter responsible]
```

Comment:
So now the role attribute of a record is an array instead of a string. Why? And are you using this factory anywhere?